programs/local/chdb.h

```diff
@@ -1,16 +1,12 @@
 #pragma once
 #include <cstdint>
 #include <stddef.h>
+#include "query_result.h"
 
 extern "C" {
 struct local_result
 {
-    char * buf;
-    size_t len;
-    void * _vec; // std::vector<char> *, for freeing
-    double elapsed;
-    uint64_t rows_read;
-    uint64_t bytes_read;
+    query_result_* result_;
```

Comment:
This will also affect libchdb users. @lmangani

Reply:
remove `query_result_` field in `struct local_result`